R/ml_generalized_linear_regression.R

```diff
@@ -88,6 +91,10 @@ ml_generalized_linear_regression <-
     invoke("setFamily", family) %>%
     invoke("setLink", link)
 
+  if (!is.null(weights_column))
+    model <- model %>%
+    invoke("setWeightCol", weights_column)
```

Comment:
The indentation here is a bit misleading -- can we just wrap the expression in braces and indent continuations for the pipe operator?